protected/humhub/modules/stream/widgets/StreamViewer.php

```diff
@@ -101,15 +103,13 @@ class StreamViewer extends JsWidget
      */
     public function init()
     {
-        if ($this->streamAction == "") {
-            throw new Exception('You need to set the streamAction attribute to use this widget!');
-        }
+        parent::init();
 
         // Setup default messages
-        if ($this->messageStreamEmpty == "") {
+        if ($this->messageStreamEmpty === '') {
```

Comment:
```php
    /**
     * @var string the path to Stream Action to use
     */
    public $streamAction = "";
```

$streamAction can not be NULL or `empty`, should be string only

Reply:
Just because its an empty string by default does not mean it can't be null, this value may be set manually perhaps by a filter or something. Don't get me wrong I don't see this as a major issue, but I don't see the advantage of removing the additional check.